blocks/metrics/prometheus.go

```diff
@@ -0,0 +1,50 @@
+package metrics
+
+import (
+	"github.com/spacemeshos/go-spacemesh/metrics"
+)
+
+const (
+	subsystem     = "blocks"
+	diffTypeLabel = "diff_type"
+)
+
+// AvgLayerBlockSize checks average block size.
+var AvgLayerBlockSize = metrics.NewHistogram(
```

Comment:
btw NewHistogram misses buckets configuration. the default buckets are tailored to latency measurements, and they need to be changed